restAPI/controllers/controller_auth.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+/* eslint-disable-next-line */
+const router = express.Router();
+/* eslint-disable-next-line */
+const logger = require("../../logger").logger;
+const { DATABASE, COLLECTION } = require("../../public/constant");
+const jwt = require("jsonwebtoken"); // import jwt to sign tokens
+
+/**
+ * @async
+ * @route   GET /api/v1/auth/testAPI
+ * @returns {object} test API
+ * @author  Hassan Ali
+ * @access  public
+ * @version 1.0
+ */
+
+exports.testAPI = async (req, res, next) => {
+  res.status(200).send({
+    error: false,
+    code: 200,
+    status: "succsess",
+    message: "Test API Works",
+  });
+};
+
+// DB connection
+const mongo_conn_native = require("../../mongo_conn_native").Connection;
+
+/**
+ * @async
+ * @route       Post /api/v1/auth/:id
+ * @returns     {Users} user object
```

Comment:
```suggestion
 * @route       Post /api/v1/auth/login
 * @returns     {JWT Token} // for example: eyJhbGciOiJIUzI1NiIsInR5cCI6Ikp
```
Comments should be up to date.